assets/js/controllers/request.controller.js

```diff
@@ -2,7 +2,7 @@ angular
   .module('walletApp')
   .controller("RequestCtrl", RequestCtrl);
 
-function RequestCtrl($scope, Wallet, Alerts, currency, $uibModalInstance, $log, destination, focus, $translate, $stateParams, filterFilter, $filter, format) {
+function RequestCtrl($rootScope, $scope, Wallet, Alerts, currency, $uibModalInstance, $log, destination, focus, hasLegacyAddress, $translate, $stateParams, filterFilter, $filter, format) {
```

Comment:
`hasLegacyAddress` needs to be present everywhere this modal is opened. That was missing in the addresses screen, so the modal broke. Fixed in c37a58032f043.